style/CONTRIBUTE.md

```diff
@@ -38,7 +40,7 @@ server which is beyond the scope of this guide.
 
 These development tools are required and must be installed manually on your system:
 
-- **NodeJS 14** or newer with **NPM 7** or newer
+- **NodeJS Node.js 17.3** or newer with **NPM 8.3.0** or newer
```

Comment:
Why are we requiring Node 17.3 now?  I'm running Node 16 and it seems to work fine on the main branch with this PR merged in.  16 is the current LTS release and even Node 14 will be in maintenance mode for a quite a while longer.
https://nodejs.org/en/about/releases/

Reply:
Just tested with Node 14 and NPM 8.5 on the openmapsamples branch and it seems to work fine.  Unless there is something that doesn't work properly with these older versions of Node I'd suggest we keep the minimum at Node 14, with the latest LTS recommended of course.